bcgov/mds
# MDS-4072: Populate dropdown list project leads

# Main

- Implement the configuration to assign project leads in party component.

# Other

- Seeded data to add all project leads provided in ticket.
- Added project leads to static data.

# How to test

- Create a user.

Access Project Lead user:
- When selecting the user, you can see Project Lead as a Role and an open ending date .
- Click on Update Contact button at the top right corner.

Deactivate Project Lead role:
- Fill the required data and disable the Project Lead role for the user.
- In history, the user with Project Lead role has an ending date of today.

Reactivate Project Lead role:
- Update again the user to activate the Project Lead role and set the start date to today.
- The user gets the Project Lead role activated again.

Assign Inspector role:
- Update the user to activate the Inspector role and set the start date to today
- The user gets both roles: Project Lead and Inspector

# Notes
https://bcmines.atlassian.net/browse/MDS-4072
